docs/reference/builder.md

```diff
@@ -28,20 +28,19 @@ Practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-pr
 ## Usage
 
 The [`docker build`](commandline/build.md) command builds an image from
-a `Dockerfile` and a *context*. The build's context is the files at a specified
+a `Dockerfile` and a *context*. The build's context is  files at a specified
```

Comment:
"The build's context _is the set of_ files at a specified location `PATH` or `URL`"